plasmapy/diagnostics/thomson.py

```diff
@@ -128,6 +129,11 @@ def spectral_density_lite(
         Defaults to [0, 1, 0] which, along with the default ``probe_vec``,
         corresponds to a 90 degree scattering angle geometry.
 
+    scattering_angle: float
```

Comment:
As written now, it looks like this function can ONLY take the scattering angle. I think that's actually the right way to do it, since otherwise you need to validate that not all three are provided and we try to avoid validation in docstrings. So, I think you should remove `scatter_vec` and `probe_vec` from the docstring and function signature here and make 'scattering_angle` a required keyword argument

Reply:
That is a great idea. Let's have this function do one thing and do it well, and the helper utility of calculating the angle from the vectors can be moved to the "top level" `spectral_density` function. Awesome!